packages/slice-machine/components/Simulator/index.tsx

```diff
@@ -16,12 +13,26 @@ import {
   selectSimulatorUrl,
 } from "@src/modules/environment";
 import { SliceMachineStoreType } from "@src/redux/type";
+import { useRouter } from "next/router";
+import { selectCurrentSlice } from "@src/modules/selectedSlice/selectors";
 
 export type SliceView = SliceViewItem[];
 export type SliceViewItem = Readonly<{ sliceID: string; variationID: string }>;
 
 export default function Simulator() {
-  const { Model, variation } = useContext(SliceContext);
+  const router = useRouter();
```

Comment:
remember to pass Router to the connected router to keep both in sync :)
https://github.com/prismicio/slice-machine/pull/472/files#diff-af0d93898897cf163cba7f962dd24f74a4a562e132bfccaf158b33ec02aa535fR114

Reply:
changed!